demo/full/scripts/controllers/Player.jsx

```diff
@@ -119,6 +121,17 @@ class Player extends React.Component {
               <div
                 className="video-screen"
                 onClick={() => this.onVideoClick()}
+                ref={
+                  (el) => {
+                    if (el) {
+                      const { offsetWidth } = el;
+                      this.state.videoScreenStyle = {
+                        height: offsetWidth / this.state.videoRatio + "px",
+                      };
+                    }
+                  }
+                }
+                style={this.state.videoScreenStyle}
```

Comment:
This blocks the video height as it was when it was first loaded. If you load the page and resize it, you have funny results.